Monika-After-Story/MonikaModDev
# Anonymioo Ribbons

#4339 

Adds ribbons from @Anonymioo 

Contains:
* Bisexual Pride Themed Ribbon
* Black and White Ribbon
* Bronze Ribbon
* Brown Ribbon
* Gradient Ribbon
* Low-Poly Gradient Ribbon
* Pink and White Striped Ribbon
* Blue and White Striped Ribbon
* Rainbow (Gradient) Ribbon
* Starry Black Sky Ribbon (Star Sky)
* Starry Red Sky Ribbon (Star Sky Variant 2)
* White Polka-dots on Red Ribbon (Red With Dots)
* Transexual Pride Themed Ribbon

Also adds handling for generic selector dialogue for things that don't have selector dialogue

TODO:
- [x] - thumbnails
- [x] - selector dlg
- [x] - `dlg_desc` (if desired)

# Testing
* verify new ribbons look ok
* verify ribbons that do not have selector dialogue pull from `generic_sel_dlg_quips`
